pnpm lockfile error after changing project package manager to pnpm

I keep getting the error message 
```
ERR_PNPM_OUTDATED_LOCKFILE  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up-to-date with package.json

Note that in CI environments this setting is true by default. If you still need to run install in such cases, use "pnpm install --no-frozen-lockfile"
```
when running `projen upgrade` after changing the project package manager to `packageManager: NodePackageManager.PNPM`. 

Tested it with a regular TS project and a TS CDK project. 
E.g. 
```
const project = new awscdk.AwsCdkTypeScriptApp({
  cdkVersion: '2.16.0',
  defaultReleaseBranch: 'main',
  name: 'test-app',
  packageManager: NodePackageManager.PNPM
});
project.synth();
```

Any ideas on how to fix this through projen?

--- a/src/javascript/node-package.ts
+++ b/src/javascript/node-package.ts
@@ -973,7 +973,9 @@ export class NodePackage extends Component {
         return frozen ? "npm ci" : "npm install";
 
       case NodePackageManager.PNPM:
-        return ["pnpm i", ...(frozen ? ["--frozen-lockfile"] : ["--no-frozen-lockfile"])].join(" ");
+        return frozen
+          ? "pnpm i --frozen-lockfile"
+          : "pnpm i --no-frozen-lockfile";
 
       default:
         throw new Error(`unexpected package manager ${this.packageManager}`);

